tests/imposter/data/questions.json

```diff
@@ -6,6 +6,12 @@
         "info": {
           "months": "3"
         }
+      },
+      {
```

Comment:
Can we rename this set of questions `single-question` -> `payslips` - will need to rename the stub API config and feature scenario tag too